hie.yaml

```diff
@@ -0,0 +1,2 @@
+cradle:
```

Comment:
Is it really needed? It doesn't list files that should be handled differently than the default behaviour of HLS. Is it only for making HLS choose cabal instead of stack?

Reply:
I will remove it, it’s just that my editor incorrectly detected this as a stack project no matter what I did 😓

Reply:
Ah ok. :) 
FYI you can ignore files locally by adding them to `.git/info/exclude` (https://luisdalmolin.dev/blog/ignoring-files-in-git-without-gitignore/)

Reply:
Oh yeah, it’s just that this is a wip and I didn’t think it would be an issue 😇